flexeval/core/language_model/openai_api.py

```diff
@@ -42,9 +43,13 @@
 def _retry_on_error(
     openai_call: Callable[[], T],
     empty_response: BaseModel,
-    max_num_trials: int = 5,
-    first_wait_time: int = 10,
+    max_num_trials: int | None = None,
+    first_wait_time: int | None = None,
+    max_wait_time: int | None = None,
```

Comment:
次のようにしてしまった方がコード量減って若干分かりやすそうです。

```suggestion
    max_num_trials: int = 5,
    first_wait_time: int = 10,
    max_wait_time: int = 80,
```

ただ `OpenAIChatAPI.__init__` でもデフォルト値の設定の重複が必要になってしまうのが、少しメンテコストを上げてしまいますね。
となると `_retry_on_error` は単体で使うことがほとんどないはずで、デフォルト値を設定するありがたみは薄いと思うので、次のように必ず明示する形もアリですね。

```suggestion
    max_num_trials: int,
    first_wait_time: int,
    max_wait_time: int,
```